packages/decision-tree/src/components/Question/components/IDontKnowButton/types.ts

```diff
@@ -0,0 +1,8 @@
+import { PaginationAction, SubmitAnswerFn } from '@hooks';
+import { Dispatch } from 'react';
+
+export type IDontKnowButtonProps = {
+  submitIDKAnswer: SubmitAnswerFn;
+  currentPage: string;
```

Comment:
what is the `currentPage` and why this component needs to know anything about it?